samples/sampler/stories/default/Image.js

```diff
@@ -1,15 +1,15 @@
 import {action} from '@enact/storybook-utils/addons/actions';
-import {object, select} from '@enact/storybook-utils/addons/controls';
+import {select} from '@enact/storybook-utils/addons/controls';
 import {mergeComponentMetadata} from '@enact/storybook-utils';
 import ri from '@enact/ui/resolution';
 import Image, {ImageBase, ImageDecorator} from '@enact/agate/Image';
 
 import css from './Image.module.less';
 
 const src = {
-	'hd':  'http://via.placeholder.com/200x200',
-	'fhd': 'http://via.placeholder.com/300x300',
-	'uhd': 'http://via.placeholder.com/600x600'
+	hd:  'http://via.placeholder.com/200x200/9037ab',
+	fhd: 'http://via.placeholder.com/300x300/9037ab',
+	uhd: 'http://via.placeholder.com/600x600/9037ab'
```

Comment:
Why we should change these  img?

Reply:
We decided to use the local image because the image does not load when traffic is exceeded.